packages/ui/src/components/Tabs/Tabs.tsx

```diff
@@ -70,61 +67,31 @@ const Tabs: React.FC<PropsWithChildren<TabsProps>> & TabsSubComponents = ({
       children?.[0]?.props?.id
   )
 
-  useEffect(() => {
```

Comment:
yeah good point, this is trying to do much for a basic tab

Reply:
alright, I've pulled it out as a HOC. Decided on a HOC because I have another PR that adds another functionality to tabs (stickiness) and I want to be able to compose the two, but open to other suggestions